internal/pkg/dependencies/project.go

```diff
@@ -121,3 +122,21 @@ func (v project) ObjectIDGeneratorFactory() func(ctx context.Context) *storageap
 		return storageapi.NewTicketProvider(ctx, v.StorageApiClient())
 	}
 }
+
+type MasterTokenRequiredError struct{}
+
+func (MasterTokenRequiredError) StatusCode() int {
+	return http.StatusUnauthorized
+}
+
+func (MasterTokenRequiredError) Error() string {
+	return "a master token of a project administrator is required"
+}
+
+func (MasterTokenRequiredError) ErrorName() string {
+	return "masterTokenRequired"
+}
+
+func (MasterTokenRequiredError) ErrorUserMessage() string {
+	return "Please provide a master token of a project administrator."
+}
```

Comment:
Ak chyba implementuje tieto metody (staci niektore), tak sa meni to, ako ku nej pristupujeme, nazvy metod by mali byt dostatocne jasne.

----

Priklad inej chyby, ktoru tam uz mame:
https://github.com/keboola/keboola-as-code/blob/74e61b60dc247d45c85578e9d70cad5161a81a65/internal/pkg/api/server/templates/http/error.go#L45-L59